examples/text_to_speech_v1.py

```diff
@@ -11,7 +11,7 @@
 print(json.dumps(text_to_speech.voices(), indent=2))
 
 with open(join(dirname(__file__), '../resources/output.wav'), 'wb') as audio_file:
-    audio_file.write(text_to_speech.synthesize('Hello world!'))
+    audio_file.write(text_to_speech.synthesize('Hello world!',accept='audio/wav',voice="en-US_AllisonVoice"))
```

Comment:
Gladly. Did you get the updates?

On Tue, Jun 28, 2016 at 1:04 PM, German Attanasio Ruiz <
notifications@github.com> wrote:

> In examples/text_to_speech_v1.py
> <https://github.com/watson-developer-cloud/python-sdk/pull/72#discussion_r68832165>
> :
>
> > @@ -11,7 +11,7 @@
> >  print(json.dumps(text_to_speech.voices(), indent=2))
> >
> >  with open(join(dirname(__file__), '../resources/output.wav'), 'wb') as audio_file:
> > -    audio_file.write(text_to_speech.synthesize('Hello world!'))
> > +    audio_file.write(text_to_speech.synthesize('Hello world!',accept='audio/wav',voice="en-US_AllisonVoice"))
>
> can we get an space between the parameters?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/watson-developer-cloud/python-sdk/pull/72/files/73b0f059c5271d83226b9c9ea0a567c8b2c93e1f#r68832165>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAxntrtt3fF3joG9dPMDTOEKl3x9JG2uks5qQX5ggaJpZM4I9-aO>
> .
>